src/bumper_emulator.cpp

```diff
@@ -11,14 +11,18 @@ class BumperEmulator
     BumperEmulator()
     {
         ros::NodeHandle nh;
+        std::string bumper_f_param, bumper_b_param;
+        if (!nh.getParam("bumper_f_", bumper_f_param)) {ROS_ERROR_STREAM("Parameter " << "bumper_f_" << " not set");};
+        if (!nh.getParam("bumper_b_", bumper_b_param)) {ROS_ERROR_STREAM("Parameter " << "bumper_b_" << " not set");};
+
```

Comment:
Just continuing in these cases is really bad.

When left empty this will just create a publisher on the namespace and might result in an error when the namespace is empty.

So either don't create the publisher when left empty OR cause the program to stop. In that case change logs to fatal.